packages/wallets/keystone/src/adapter.ts

```diff
@@ -0,0 +1,143 @@
+
+    private _keyring: DefaultKeyring | null;
+    private _publicKey: PublicKey | null;
+    private _connecting: boolean;
+    private _readyState: WalletReadyState = WalletReadyState.Installed;
+
+    constructor(config: KeystoneWalletAdapterConfig = {}) {
+        super();
+        this._keyring = null;
+        this._publicKey = null;
+        this._connecting = false;
+    }
+
+    async connect(): Promise<void> {
+        try {
+            if (this.connected || this.connecting) return;
```

Comment:
Can you format all of this with prettier?  It's hard to review because the indentation, brackets, and semicolons are all over the place. `yarn run fmt` from the project root should do the trick.